server/tables/tables.go

```diff
@@ -322,14 +322,14 @@ type Execution struct {
 	Worker      string
 	// Command Snippet
 	CommandSnippet          string
-	InvocationID            string `gorm:"index:execution_invocation_id"`
+	InvocationID            string `gorm:"index:executions_invocation_id_stage"`
```

Comment:
we'll have to watch this during the release, this is our biggest table

Reply:
Yeah, it's just slightly scary but should be fine. I will keep an eye on it during the release. I plan to merge this after tomorrow's release to give this at least a week to soak in dev.